modules/library/metadata/src/main/java/org/geotools/measure/BaseUnitFormatter.java

```diff
@@ -74,28 +79,6 @@ public BaseUnitFormatter(List<UnitDefinition> unitDefinitions) {
         }
     }
 
-    /**
-     * @return an immutable map that shows the units (associated with their symbols) that this unit
-     *     formatter contains
-     */
-    public Map<Unit<?>, String> getUnitToSymbolMap() {
-        return Collections.unmodifiableMap(this.unitToName);
-    }
-
-    /**
-     * @return an immutable map that shows the symbols (associated with their units) that this unit
-     *     formatter contains
-     */
-    public Map<String, Unit<?>> getSymbolToUnitMap() {
-        return Collections.unmodifiableMap(this.nameToUnit);
-    }
```

Comment:
Public API in a public class cannot be removed like this. It has to be kept functional, and deprecated, for a release cycle, and only then it can be removed.
The change should also be noted in the upgrade guide here: https://github.com/geotools/geotools/blob/main/docs/user/welcome/upgrade.rst

Reply:
I thought it was only used by tests and therefore not necessary. I put them back